src/runtime/node/diagnostics_channel/index.ts

```diff
@@ -0,0 +1,182 @@
+import noop from "../../mock/noop";
+import diagnostics_channel from "node:diagnostics_channel";
```

Comment:
hmm.. I'm not sure how you intended this to work but I *think* this import will point back at this file resulting in an infinite loop.

Reply:
Yup, this was supposted to be `import type diagnostics_channel from "node:diagnostics_channel";`, which I've updated it to.